CMakeLists.txt

```diff
@@ -3,10 +3,12 @@ project(rtrlib)
 cmake_minimum_required(VERSION 2.6)
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
-
+set(NOSSH false)
```

Comment:
On the contrary, I think it does address all points (and concerns) in #47. Further, it is very unlikely, that ```libssh``` is not found _after_ Travis was told to install it explicitly ;-)